wicket-core/src/main/java/org/apache/wicket/markup/head/ResourceAggregator.java

```diff
@@ -353,6 +353,7 @@ private void renderCombinedEventScripts()
 		}
 		if (combinedScript.length() > 0)
 		{
+			combinedScript.append("\nWicket.Event.publish(Wicket.Event.Topic.AJAX_TIMERS_BOUND);");
```

Comment:
Also, I noticed that it doesn't work for header items added via the `AjaxRequestTarget`.
Should that be a concern?